website/static/js/institutionProjectSettings.js

```diff
@@ -165,7 +174,9 @@ var ViewModel = function(data) {
             else {
                 index = self.affiliatedInstitutionsIds().indexOf(item.id);
                 var removed = self.affiliatedInstitutions.splice(index, 1)[0];
-                self.availableInstitutions.push(removed);
+                if (self.userInstitutionsIds.indexOf(removed.id) > -1) {
```

Comment:
Won't this always be false? IIRC, `indexOf` returns `-1` if the item isn't in the list.